vscode_extension/src/extension.ts

```diff
@@ -7,6 +7,7 @@ import { setLogLevel } from "./commands/setLogLevel";
 import { showSorbetActions } from "./commands/showSorbetActions";
 import { showSorbetConfigurationPicker } from "./commands/showSorbetConfigurationPicker";
 import { toggleUntypedCodeHighlighting } from "./commands/toggleUntypedCodeHighlighting";
+import { savePackageFiles } from "./commands/savePackageFiles";
```

Comment:
Nit: project file imports are sorted by import path, can this be moved to line 6?